packages/contracts-bedrock/deploy-config/hardhat.json

```diff
@@ -38,6 +39,11 @@
   "numDeployConfirmations": 1,
   "eip1559Denominator": 50,
   "eip1559Elasticity": 10,
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflict is pending here.

Reply:
It's very odd that I haven't been able to find these when searching in VSCode... I ended up having to use ripgrep to find them.